andrea/js/app.js

```diff
@@ -0,0 +1,74 @@
+// buttons //
+let signUpButton = document.querySelector("#navbar-sign");
+let logInButton = document.querySelector("#navbar-login");
+
+let createButton = document.querySelector(".create-btn")
+
+let welcomeView = document.querySelector(".welcome-view")
+
+// form //
+let form = document.querySelector(".register-form");
+let registerTitle = document.querySelector(".register-title");
+let logInTitle = document.querySelector(".login-title");
+let registerFields = document.querySelectorAll(".register-fields");
+
+let logInForm = document.querySelector(".login-form");
+let formAlternative = document.querySelector(".form-alternative");
+
+// passwords //
+let password = document.getElementById("password");
+let confirmPassword = document.getElementById("confirm-password");
+
+// show register form //
+function showRegister() {
+    welcomeView.classList.add("hidden");
+    signUpButton.classList.add("hidden");
+    logInButton.classList.add("hidden");
+    form.classList.add("form__show");
```

Comment:
I think you can remove the `__show` part of this classname for 2 reasons. The first one, the default state of the form is shown, so no need for redundancy by adding the `show` word on the name, but IF you still need to have a class for when the form is not hidden, then consider that `hidden` and its opposite is a MODIFIER or state, therefore, according to BEM, it needs to be added to the classname using a double dash, like `form--visible` (visible is more common that `show` for your particular use case)